core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -289,6 +306,15 @@ private CallOptions() {
   }
 
   /**
+   * Returns whether the call is
+   * <a href="https://github.com/grpc/grpc/blob/master/doc/fail_fast.md">fail fast</a>.
+   * Fail fast is the default option for gRPC calls.
+   */
+  public boolean isFailFast() {
```

Comment:
`@ExperimentalApi`

It seems strange this doesn't match the `with*`. Name it `isWaitForReady`?